Ccq/etc/queue_publisher.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework-message-queue:etc/publisher.xsd">
+    <publisher topic="rltsquare">
+        <connection name="amqp" exchange="magento"/>
```

Comment:
Unnecessary/inappropriate code - specifying connection name ...  good intention, not really necessary